superchain/chain.go

```diff
@@ -117,29 +137,49 @@ func (c *Chain) populateGenesis() {
 
 func init() {
 	var err error
-	configDataReader, err = zip.NewReader(bytes.NewReader(configData), int64(len(configData)))
+	BuiltInConfigs, err = NewChainConfigReader(builtInConfigData)
 	if err != nil {
-		panic(fmt.Errorf("opening zip reader: %w", err))
+		panic(err)
+	}
+	Chains = BuiltInConfigs.Chains
+}
+
+func NewChainConfigReader(configData []byte) (*ChainConfigLoader, error) {
```

Comment:
Nit: could we harmonize the naming (`ChainConfigReader` vs `ChainConfigLoader`) and put this constructor nesxt to the struct definition?